Gemfile.lock

```diff
@@ -1,3 +1,11 @@
+GIT
```

Comment:
Regenerate this file again as per above changes.

Reply:
Done